desktop/ui/src/main/java/org/datacleaner/guice/DCModuleImpl.java

```diff
@@ -300,7 +300,10 @@ public final DataCleanerConfiguration getDataCleanerConfiguration(@Undecorated D
                     final StorageProvider storageProvider = c.getEnvironment().getStorageProvider();
 
                     final TaskRunner taskRunner = c.getEnvironment().getTaskRunner();
-                    final RemoteServerConfiguration remoteServerConfiguration = new MutableRemoteServerConfigurationImpl(c.getEnvironment().getRemoteServerConfiguration().getServerList(), configurationWriter);
+                    final RemoteServerConfiguration remoteServerConfiguration =
+                            new MutableRemoteServerConfigurationImpl(
+                                    c.getEnvironment().getRemoteServerConfiguration().getServerList(),
+                                    c.getEnvironment().getTaskRunner(), configurationWriter);
```

Comment:
Why do `c.getEnvironment().getTaskRunner()`, when in the line above (302), it is already retrieved and assigned to a variable?